src/Kotlin/KotlinLanguageService.kt

```diff
@@ -171,24 +164,46 @@ class KotlinLanguageService : LanguageService {
         }
     }
 
+    private fun ContentNode.renderAnnotationsForNode(node: DocumentationNode) {
+        node.annotations.forEach {
+            renderAnnotation(it)
+        }
+    }
+
+    private fun ContentNode.renderAnnotation(node: DocumentationNode) {
```

Comment:
inline options should ideally be translated to some human-readable note/text in the docs. It's important to understand what's being inlined in performance-critical code.